test_runner/regress/test_timeline_archive.py

```diff
@@ -1,27 +1,25 @@
 import pytest
 from fixtures.common_types import TenantId, TimelineArchivalState, TimelineId
 from fixtures.neon_fixtures import (
-    NeonEnv,
+    NeonEnvBuilder,
 )
 from fixtures.pageserver.http import PageserverApiException
 
 
-def test_timeline_archive(neon_simple_env: NeonEnv):
-    env = neon_simple_env
```

Comment:
Just out of interest, why were you initially using neon_simple_env?

Reply:
When I copy pasted the original from `test_timeline_delete`, I didn't feel like it needed a modification.

Reply:
I am still hoping to nuke the whole `neon_simple_env` so it was interesting to know.